dev/tools/grunt/tasks/static.js

```diff
@@ -26,7 +26,11 @@ module.exports = function (grunt) {
 
         setConfig('eslint', currentTarget, cvf.getFiles(file));
         setConfig('jscs', currentTarget, cvf.getFiles(file));
-        grunt.option('force', true);
+
+        if (grunt.option('force') !== false) {
```

Comment:
Please, revert this changes, we already solve this issue. We add [grunt-continue](https://www.npmjs.com/package/grunt-continue), that provide ability to run in force mode(for collect eslint and jscs errors) and fail on warnings. Internal ticket MAGETWO-67342 now in delivery process.